language/backend/en-GB/en-GB.com_j2store.ini

```
@@ -118,21 +118,21 @@ J2STORE_ADVANCED_SETTINGS="Advanced Settings"
 J2STORE_INTEGRITY_CHECK="Integrity Check"
 J2STORE_CONGRATULATIONS="Congratulations!"
 J2STORE_POSTCONFIG_WELCOME_MESSAGE="You are just a few steps away from building your online shop with J2Commerce"
-J2STORE_POSTCONFIG_WHATTHIS="This wizard will help you configure the basic settings. <br>These can be managed anytime at J2Store -> Setup -> Configuration."
-J2STORE_POSTCONFIG_WHATTHIS_LINKTEXT="J2Store -> Configuration"
+J2STORE_POSTCONFIG_WHATTHIS="This wizard will help you configure the basic settings. <br>These can be managed anytime at J2Commerce -> Setup -> Configuration."
+J2STORE_POSTCONFIG_WHATTHIS_LINKTEXT="J2Commerce -> Configuration"
 J2STORE_INTEGRITY_CACHE_PLUGIN_ENABLED="System - Cache plugin is ENABLED. Recommended Setting: DISABLED"
 J2STORE_INTEGRITY_CACHE_PLUGIN_LABEL="System - Cache Plugin"
-J2STORE_INTEGRITY_SYSTEM_PLUGIN_LABEL="System - J2Store Plugin"
-J2STORE_INTEGRITY_SYSTEM_PLUGIN_ENABLED="System - J2Store plugin is ENABLED."
+J2STORE_INTEGRITY_SYSTEM_PLUGIN_LABEL="System - J2Commerce Plugin"
+J2STORE_INTEGRITY_SYSTEM_PLUGIN_ENABLED="System - J2Commerce plugin is ENABLED."
 
-J2STORE_POSTCONFIG_STORE_SETUP_DONE_ALREADY="First run wizard was already executed. If you need to alter the settings, please go to J2Store -> Configuration"
+J2STORE_POSTCONFIG_STORE_SETUP_DONE_ALREADY="First run wizard was already executed. If you need to alter the settings, please go to J2Commerce -> Configuration"
 
 J2STORE_DEFAULT_TAX_RATE="Default Tax Rate (optional)"
 J2STORE_DEFAULT_TAX_RATE_DESC="You can choose to create a default tax rate. You can also do this after completing this wizard. This is purely optional"
 J2STORE_POSTCONFIG_LBL_MANDATORYINFO="Free Version License / Support"
 J2STORE_AGREE_SET_UP_STORE="Agree & Setup Store"
 J2STORE_POSTCONFIG_LBL_ACCEPTLICENSE="I have read the license, I have understood the licence, and by clicking here I accept the licence"
-J2STORE_POSTCONFIG_DESC_ACCEPTLICENSE="J2Store is distributed under the terms of the <a target="_QQ_"_blank"_QQ_" href="_QQ_"http://www.gnu.org/licenses/gpl.html"_QQ_">GNU General Public License (GPL)</a>, version 3 of the license or –at your option– any later version published by the Free Software Foundation. It is the same license Joomla!&trade; itself is licensed under. You have to accept it if you want to use this component."
+J2STORE_POSTCONFIG_DESC_ACCEPTLICENSE="J2Commerce is distributed under the terms of the <a target="_QQ_"_blank"_QQ_" href="_QQ_"http://www.gnu.org/licenses/gpl.html"_QQ_">GNU General Public License (GPL)</a>, version 3 of the license or –at your option– any later version published by the Free Software Foundation. It is the same license Joomla!&trade; itself is licensed under. You have to accept it if you want to use this component."
```

```suggestion
J2STORE_POSTCONFIG_DESC_ACCEPTLICENSE="J2Commerce is distributed under the terms of the <a target=\"_blank\" href=\"http://www.gnu.org/licenses/gpl.html\">GNU General Public License (GPL)</a>, version 3 of the license or –at your option– any later version published by the Free Software Foundation. It is the same license Joomla!&trade; itself is licensed under. You have to accept it if you want to use this component."
```

we dont use QQ anymore just escape the "

---

Thanks Brian!